pages/timeseries.js

```diff
@@ -465,6 +468,7 @@ export default class TimeSeries extends Component {
                     className='basic-multi-select'
                     classNamePrefix='select' />
                 </fieldset>
+                <input className='button button--primary' type='button' value='Query' onClick={this.updateTimeseriesData} />
```

Comment:
```suggestion
                <input className='button button--primary' type='submit' value='Submit' onClick={this.updateTimeseriesData} />
```
@kamicut should this button type be a submit for the form? Regardless, I think "Submit" could be a better button value text here.